webapp-django/crashstats/signature/templates/signature/signature_report.html

```diff
@@ -145,8 +184,12 @@
 <script>
 var BASE_URL = location.protocol + '//' + location.host;
 var FIELDS = {{ fields | json_dumps }};
+var DEFAULT_CHANNEL = {{ channels[3] | json_dumps }};
```

Comment:
This is bad: if a user changes the configuration, you cannot be assured that ``channels[3]`` is ``nightly``. You'll need to create a different settings option to have that data. 

(In a previous comment I said you could use ``channels[0]``. That's fine to do because we often consider that the first element of a list is the default value. However here it seems ``nightly`` is not the first element, and should not be, so it was just a bad suggestion on my part. )